src/components/AssetDetails/Description.js

```diff
@@ -0,0 +1,45 @@
+import { render } from '../../shared/dom';
+
+/**
+ * @param {object} movie
+ * @param {string} name
+ * @param {string} index
+ * @returns {string}
+ */
+const DescriptionRow = (movie, name, index) => {
+    if (!index) index = name.toLowerCase();
+
+    return `
+        <tr>
```

Comment:
Using tables is really oldschool and now it has only use case in things such as displaying financial data etc.

We don't have such a use case here, in case od any display changes etc we would have to delete most of the code

Use just divs and flexbox for ex.